jwql/website/apps/jwql/data_containers.py

```diff
@@ -496,7 +497,7 @@ def get_expstart(instrument, rootname):
     return expstart
 
 
-def get_filenames_by_instrument(instrument, restriction='all'):
+def get_filenames_by_instrument(instrument, restriction='all', query_file=None, query_response=None):
     """Returns a list of filenames that match the given ``instrument``.
 
     Parameters
```

Comment:
@bhilbert4 what is this query file parameter? Would you be able to update the docstring here?